Utils/Functions/Net/GithubReleases/script.js

```diff
@@ -1,29 +1,41 @@
 const Downloader = include("utils.functions.net.download");
-const {createTempDir, remove, cat} = include("utils.functions.filesystem.files");
+const Resource = include("utils.functions.net.resource");
+const { createTempDir, remove, cat } = include("utils.functions.filesystem.files");
 
 /**
- * Get an array of tag corresponding to the releases in a git repository
+ * Fetches an array of tag corresponding to the releases in a git repository
  *
  * @param {string} repositoryOwner owner of the repository
  * @param {string} repositoryName name of the repository
  * @param {wizard} wizard the wizard
- * @returns {string[]} list of version tag in the repository
+ * @returns {object[]} list of version objects in the repository
  */
-module.getGithubReleases = function getGithubReleases(repositoryOwner, repositoryName, wizard) {
+module.fetchGithubReleases = function fetchGithubReleases(repositoryOwner, repositoryName, wizard) {
```

Comment:
I have renamed the method to `fetchGithubReleases` because I think it is more fitting.